CODEOWNERS

```diff
@@ -38,7 +38,19 @@ NOTICES     @garyverhaegen-da @aherrmann-da @cocreature @dasormeter
 /ledger/      @gerolf-da @mziolekda @nmarton-da @rautenrieth-da @tudor-da
 /ledger-api/  @gerolf-da @mziolekda @nmarton-da @rautenrieth-da @tudor-da
 
-/ledger/participant-state/kvutils/ @fabiotudone-da @gerolf-da @hubert-da @miklos-da @nicu-da @SamirTalwar-DA
+/ledger/ledger-api-common/          @fabiotudone-da @gerolf-da @hubert-da @miklos-da @nicu-da @mziolekda @nmarton-da @rautenrieth-da @SamirTalwar-DA @tudor-da
```

Comment:
> I created two teams. could you use those instead of the individual names?

Awesome, will do once we agree on the below.

> Also please keep in mind that more specific folder definitions override more generic ones, so you'll have to add both teams to retain the original notification structure.

Yes, that was intentional. Do you think @digital-asset/kv-participant should be notified about the folders I removed them from?

They are:

- ledger-on-memory
- ledger-on-sql
- Sandbox (Next), which is basically ledger-on-sql
- the participant state APIs

Let me know to which directories I should re-add them, and I'll do that.

Reply:
FWIW, I would like to be notified about changes to the participant state API because we use the API in the indexer/api server.

Reply:
@rautenrieth-da, good to know, I hadn't thought about that!

Do you want notifications on Sandbox changes too?

Reply:
Sandbox classic yes, sandbox next not necessarily.

Reply:
Great. I have left Sandbox Classic under the participant team purview because it's so intertwined with the index. I'll make the changes to the participant state APIs.

Reply:
Done.